src/OSPSuite.Infrastructure.Import/Core/DataFormat/AbstractColumnsDataFormat.cs

```diff
@@ -273,7 +274,10 @@ private SimulationPoint parseMappingOnSameColumn(MappingDataFormatParameter curr
             };
          if (element.StartsWith("<"))
          {
-            double.TryParse(element.Substring(1), out result);
+            var separatorChar = Convert.ToChar(CultureInfo.CurrentCulture.NumberFormat.NumberDecimalSeparator);
+            
+            var lloqString = element.Substring(1).Replace(',', separatorChar).Replace('.',separatorChar);
+            double.TryParse(lloqString, out result);
```

Comment:
Isn't it weird to not verify that you can parse? What happens if this does not work? 
Also we have a method that converts a string to double 
https://github.com/Open-Systems-Pharmacology/OSPSuite.Utility/blob/master/src/OSPSuite.Utility/Extensions/ConversionExtensions.cs#L31

can this be used?

Reply:
I was not aware of this function. Yes this is a better alternative, since it would fill the lloq value with NaN if it cannot be parsed.  In the current way the function is usd this is not a possibility, since the values of each column are checked in the beginning and if they are not numeric you cannot map them to measurement or error. But still it does not hurt to have even this possibility covered within the function.